### DISCUSSION TITLE
private profile feature has a very bad implementation(suggestion with fix)

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [community/community#39558] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In **Settings > Public Profile** you can choose to make your profile private 

<img width="823" alt="Screenshot 2022-11-20 at 11 48 39 AM" src="https://user-images.githubusercontent.com/8083613/202888914-bc018925-f86c-48a7-b79a-cb64d8e5e1bc.png">

but you can't see someone's followers and sponsors. Instead what it should look like is that you can requests someone to follow and the person(which is me) can decide to approve or reject the request. Neither can you see his sponsors or sponsor him. That's not how privacy feature works 👎🏻 

It should look something like this. In picture below:

<img width="1086" alt="Screenshot 2022-11-20 at 11 54 04 AM" src="https://user-images.githubusercontent.com/8083613/202889205-f1eb377c-06d3-4c6e-b63a-e0b1a7992874.png">

Currently in this state private profile feature is basically unusable and you might end up turning it off because what's the point? If takes away everything from you on the name of **_privacy_** i.e. basically restricts you from growing

**Comment by jfullstackdev** [comment]
there is another suggestion for setting that to privacy, like, you have the ability to select what can be displayed but again, private profile was just recently added too
